internal/app/tfsec/custom/loader.go

```diff
@@ -0,0 +1,82 @@
+package custom
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+type ChecksFile struct {
+	Checks []*Check `json:"checks"`
+}
+
+func Load(rootPath string) error {
+	customCheckDir := fmt.Sprintf("%s/.tfsec", rootPath)
+	_, err := os.Stat(customCheckDir)
+	if os.IsNotExist(err) {
+		return nil
+	}
+
+	return loadCustomChecks(customCheckDir)
+}
+
+func loadCustomChecks(customCheckDir string) error {
+	files, err := listFiles(customCheckDir, ".*_tfchecks.json")
+	if err != nil {
+		return err
+	}
+	var errorList []string
+	for _, file := range files {
+		checkFilePath := path.Join(customCheckDir, file.Name())
+		checks, err := loadCheckFile(checkFilePath)
+		if err != nil {
+			errorList = append(errorList, err.Error())
```

Comment:
```suggestion
			errorList = append(errorList, err.Error())
			continue
```
I think if it errors we don't want to process checks that came back?